src/downloader.py

```diff
@@ -1,12 +1,10 @@
 import os
 import urllib.request
 
+
 def __get_output_file_path(base_dir: str, registry_name: str, package_name: str, version_name: str, package_ext: str) -> str:
     package_dir = os.path.join(base_dir, registry_name, package_name)
-    try:
-        os.makedirs(package_dir)
-    except:
-        pass
+    os.makedirs(package_dir)
```

Comment:
This throws irrelevant errors. Not ignoring them breaks the flow

Reply:
fixed by `os.makedirs(package_dir, mode=666, exist_ok=True)`, if a directory already exists it won't throw an error + dir permission will be `drw-rw-rw-` which should not result with an error.